Cargo.toml

```diff
@@ -75,6 +75,7 @@ rayon                 = { version = "1.9.0", default-features = false }
 serde_bytes           = { version = "0.11.12", default-features = false }
 serde_json            = { version = "1.0.108", default-features = false }
 serde                 = { version = "1.0.190", default-features = false }
+strum                 = { version = "0.26.2", default-features = false }
```

Comment:
Used by `crate::Status` (and probably other stuff in the future).

Not strictly necessary but it provides macros for things we would otherwise implement manually (`as_ref()`, custom serialization, etc), alongside other stuff.

Reply:
I'm not completely against this if you think it's worth it, however at the moment it's only used on one type

Reply:
Added a [TODO](https://github.com/Cuprate/cuprate/pull/147/commits/4830a261807ea149760039897ec36ac95aaeadc5), I'll remove it if it isn't used elsewhere by the end of RPC impl.

Reply:
Actually, removed this for now. Will add if needed.